pkgs/by-name/re/rerun/package.nix

```diff
@@ -105,13 +101,8 @@ rustPlatform.buildRustPackage rec {
 
   postPhases = lib.optionals stdenv.hostPlatform.isLinux [ "addDlopenRunpathsPhase" ];
 
-  cargoTestFlags = [
-    "-p"
-    "rerun"
-    "--workspace"
-    "--exclude=crates/rerun/src/lib.rs"
-  ];
-
+  # The path in `build.rs` is wrong for some reason, so we patch it to make the passthru tests work
+  patches = [ ./passthru.patch ];
```

Comment:
Not sure "passthru" is the right name for this, as it's a Nixpkgs concept that's something to do with the package.  Maybe "tests.patch"?

Also, if this is wrong, should it be fixed upstream?

Reply:
Yeah makes, I [created an issue upstream](https://github.com/rerun-io/rerun/issues/7411) to let them decide how to handle that. Maybe there's another workaround.